guides/source/configuring.md

```diff
@@ -260,6 +266,26 @@ All these configuration options are delegated to the `I18n` library.
 
 * `config.i18n.load_path` sets the path Rails uses to look for locale files. Defaults to `config/locales/*.{yml,rb}`.
 
+* `config.i18n.fallbacks` sets fallback behavior for missing translations. Here are 3 usage examples for this option:
+
+  * You can set the option to `true` for using default locale as fallback, like so:
+
+    ```ruby
+    config.i18n.fallbacks = true
+    ```
+
+  * Or you can set an array of locales as fallback, like so:
+
+    ```ruby
+    config.i18n.fallbacks = [:tr, :en]
+    ```
+
+  * Or you can set different fallbacks for locales individually. For example, if you want to use `:tr` for `:az` and `:de`, `:en` for `:da` as fallbacks, you can do it, like so:
+
+    ```ruby
+    config.i18n.fallbacks = { az: :tr, da: [:de, :en] }
```

Comment:
can you specify 
`config.i18n.fallbacks.map = { az: :tr, da: [:de, :en] }`
which has the same effect.
 ```ruby
config.i18n.fallbacks = { az: :tr, da: [:de, :en] } #or
config.i18n.fallbacks.map = { az: :tr, da: [:de, :en] }
```